src/map/skill.cpp

```diff
@@ -11090,7 +11088,7 @@ static int8 skill_castend_id_check(struct block_list *src, struct block_list *ta
 			case AB_CLEARANCE:
 				return USESKILL_FAIL_TOTARGET;
 			default:
-				return USESKILL_FAIL_LEVEL;
+				return USESKILL_FAIL_MAX;
```

Comment:
Not sure if this changes other intended behaviors too, but in Aegis damage skills casted on players on non-pvp maps should not show any message.